tests/time_tests/src/timetests/timetest_infer.cpp

```diff
@@ -16,40 +16,28 @@ using namespace InferenceEngine;
  * main(). The function should not throw any exceptions and responsible for
  * handling it by itself.
  */
-int runPipeline(const std::string &model, const std::string &device, const bool performanceHint,
-                const bool isCacheEnabled, const std::string &vpuCompiler) {
-  auto pipeline = [](const std::string &model, const std::string &device, const bool performanceHint,
-                     const bool isCacheEnabled, const std::string &vpuCompiler) {
+int runPipeline(const std::string &model, const std::string &device, const bool isCacheEnabled) {
+  auto pipeline = [](const std::string &model, const std::string &device, const bool isCacheEnabled) {
     Core ie;
     CNNNetwork cnnNetwork;
     ExecutableNetwork exeNetwork;
     InferRequest inferRequest;
     size_t batchSize = 0;
 
-    // set config for VPUX device
-    std::map<std::string, std::string> vpuConfig = {};
-    if (vpuCompiler == "MCM")
-      vpuConfig = {{"VPUX_COMPILER_TYPE", "MCM"}};
-    else if (vpuCompiler == "MLIR")
-      vpuConfig = {{"VPUX_COMPILER_TYPE", "MLIR"}};
-
     // first_inference_latency = time_to_inference + first_inference
     {
       SCOPED_TIMER(time_to_inference);
       {
         SCOPED_TIMER(load_plugin);
-        if (performanceHint) {
-          TimeTest::setPerformanceConfig(ie, device);
-        }
+        TimeTest::setPerformanceConfig(ie, device);
```

Comment:
all configurations should be out of timer "load_plugin" (after the timer)

Reply:
done